### ISSUE TITLE
Can i use nlp4j with spanish?

### ISSUE THREAD

**Issue by cnarvaa** [emorynlp/nlp4j-old#13]
Hello,
I was thinking about using this library in a research project, the fact is that the project is oriented to work with spanish words. Is there any way to use the library with spanish?.

Tanks. 😄

**Comment by jdchoi77** [comment]
Yes, our tools are data-driven so it should work for Spanish as well.  Please take a look at our training tutorial and let me know if you have any questions.  Thanks.
